app/components/primer/alpha/dialog/header.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Primer
+  module Alpha
+    class Dialog
+      # A `Dialog::Header` is a compositional component, used to render the
+      # Header of a dialog. See <%= link_to_component(Primer::Alpha::Dialog) %>.
+      class Header < Primer::Component
+        # @param title [String] The title of the dialog.
+        # @param subtitle [String] The subtitle of the dialog. This will also set the `aria-describedby` attribute.
+        # @param hide_divider [Boolean] If true the visual dividing line between the body and footer will be hidden
+        # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
+        def initialize(
+          id:,
+          title:,
```

Comment:
The `title` size option is another one that folks are using in dotcom (we have a default and a `large` title size). We were thinking it might make more sense to ship a `ConfirmationDialog` with that large title by default, and perhaps a bit more prescriptive about content? Thinking large title that must be visible, and two default buttons in the footer (danger and cancel). Regardless, should we add the title size back here in the meantime?

Reply:
> ConfirmationDialog

This was absolutely my intent as a fast-follow to Dialog.